nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/JoltTransformJSON.java

```diff
@@ -140,6 +141,27 @@
             .defaultValue("false")
             .build();
 
+    public static final PropertyDescriptor INPUT_CHARSET = new PropertyDescriptor.Builder()
+            .name("input-charset")
+            .displayName("Input Character Set")
+            .description("The encoding character set for the input JSON for transformation")
+            .required(true)
+            .allowableValues(StandardCharsets.UTF_8.name(), StandardCharsets.UTF_16.name(),
```

Comment:
Will allowable charset options be the same for input/output properties?  If so perhaps this could be a list shared between the two for simplicity (I believe the builder makes internal copies of the values anyway for the property descriptor instance).

Reply:
I agree for the most part as most user's aren't going to change the encoding during a Jolt Transformation, I only added the output charset because the ticket requested it.